test/integration/router_test.go

```diff
@@ -233,6 +233,12 @@ func createAndStartRouterContainer(dockerCli *dockerClient.Client, masterIp stri
 		},
 	}
 
+	err = pullIfNotPresent(dockerCli, getRouterImage())
```

Comment:
I think it's ok if it fails hard, and it's clear why.

----- Original Message -----
> > @@ -233,6 +233,12 @@ func createAndStartRouterContainer(dockerCli
> > *dockerClient.Client, masterIp stri
> >  		},
> >  	}
> >  
> > +	err = pullIfNotPresent(dockerCli, getRouterImage())
> 
> It was failing hard.  I added this in so it wouldn't.  If we're ok with the
> failure then this PR can be closed.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/879/files#r24108382